app/src/main/kotlin/io/intrepid/fablekotlin/screens/homescreen/HomescreenActivity.kt

```diff
@@ -20,14 +19,12 @@ class HomescreenActivity : BaseMvpActivity<HomescreenContract.Presenter>(), Home
     override fun onViewCreated(savedInstanceState: Bundle?) {
         super.onViewCreated(savedInstanceState)
 
-        if ((!getLoggedInStatus()) || SharedPreferencesManager.getInstance(this).token == null) run {
+        if (!getLoggedInStatus()) run {
```

Comment:
what does if/run do?

Reply:
I'm realizing looking further into this that I don't need the run. This is one of the lines that I actually just brought it over from the java project and converted and it added the run itself, but it's not actually needed. I admittedly didn't question it because it worked and looking at it I thought it made sense at the time. hmm

It looks like run is mostly to run a function block and return the result, so it didn't really stop my code from working, but wasn't necessary, I've removed it and it's working fine still.

I'll look into the other comments you left, they're great points. I molded my code a bit too much around how we originally did it.